onnxruntime/test/python/onnxruntime_test_python_iobinding.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
```

Comment:
i guess this test is being run manually?

Reply:
fixed